src/main/java/taxi/service/impl/CarServiceImpl.java

```diff
@@ -22,7 +23,7 @@ public Car create(Car car) {
     @Override
     public Car get(Long id) {
         return carDao.get(id).orElseThrow(() ->
-                new RuntimeException("Car with id " + id + " not found"));
+                new NoSuchElementException("Car with id " + id + " not found"));
```

Comment:
just a suggestion: maybe it's better not to catch runtime exceptions but to process Optional with `isPresent`?

Reply:
How?

Reply:
In service